src/gateway/Gateway.sol

```diff
@@ -89,11 +106,24 @@ contract Gateway is Auth, IGateway {
             manager = address(root);
         } else {
             // Dynamic path for other managers, to be able to easily
-            // extend functionality of Liquidity Pools
+            // extend functionality of Liquidity Poolsif(msg.sender == address(mana))
```

Comment:
Why this comment change?

Reply:
Because I have fat fingers and I might have cp'ed something at that line :rofl: